services/merak-compute/compute.go

```diff
@@ -46,7 +46,7 @@ func main() {
 
 	log.Printf("Starting gRPC server. Listening at %v", lis.Addr())
 	if err := gRPCServer.Serve(lis); err != nil {
-		log.Fatalf("failed to serve: %v", err)
+		log.Printf("failed to serve: %v", err)
 	}
```

Comment:
@yanmo96 Why you need to modify merak-compute's file?

Reply:
mistake, changed it back